### ISSUE TITLE
Missing tags for versions in PyPI?

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1107]
I was trying to debug inside a container a version of Autosubmit I installed from PyPI.

The version is 4.0.84, available here: https://pypi.org/project/autosubmit/4.0.84/

These are the tags that we have: https://earth.bsc.es/gitlab/es/autosubmit/-/tags?sort=version_desc

We don't have 4.0.84, nor v4.0.84, which is a requirement for **traceability**. There's a tag `v4.0.0b22`, which matches our version in the lua module in our HPC.

However, that probably does not match the version in LUMI, Kit, EDITO, etc.

I can try to debug the issue some other way (updating to the latest so I can sync the code with the IDE). But any clue where can I find the exact tag for our release, please @mcastril / @dbeltrankyl ?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 7, 2023, 10:59

I think we decided to match the pip version and get rid of bXX starting from next tag, no? ( in an Autosubmit meeting)

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 7, 2023, 11:00

And well answering the question, 4.0.84 should be the 4.0.0b22 one

**Comment by kinow** [comment]
Yes, but that was related to our HPC modules. I didn't remember/wasn't aware that we also didn't have tags for each release.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 7, 2023, 11:07

Ah, it is the same. Each tag corresponds to each module version in the hpc. 

The way of updating autosubmit is to upload a new tag and then do the module installation from that tag. ( previously, we were reusing the same tag )

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 7, 2023, 11:08

The last 22 updates of pypi correspond to a new tag, but the next tag release will be v4.0.85 instead of 4.0.0b23. So we won't have macth issues anymore

**Comment by kinow** [comment]
>The last 22 updates of pypi correspond to a new tag

Sorry Dani, I didn't follow. The last 22 releases to PyPI each correspond to a new tag in our GitLab, or to the same tag? For example, could you point to what are the tags for

- 4.0.79 (May 8, 2023)
- 4.0.80 (Jun 30, 2023)
- 4.0.81 (Jul 20, 2023)
- 4.0.82 (Jul, 20, 2023)
- 4.0.84 (Jul 21, 2023)

**Comment by kinow** [comment]
In GitLab by @mcastril on Aug 31, 2023, 10:38

@dbeltrankyl can you check this one?

**Comment by dbeltrankyl** [comment]
Closing this, with the "new" release standard, we no longer miss deployments.
